website/templates/project/project_base.mako

```diff
@@ -14,6 +14,11 @@
 
 <%include file="project_header.mako"/>
 
+% if status and node:
+    ## WARNING: Need new heading alert messages? They must be added at both here and base.mako
```

Comment:
How about changing the message into `## WARNING: Need display-everywhere heading messages? They must be added at both here and project_base.mako`